### PR TITLE
Preklad Hanoi

### PR DESCRIPTION
**Pull request by Benko11**
very nice

### CODE REVIEW

File: hanoi.c [hunk 1]
```diff
@@ -3,11 +3,11 @@
 void hanoi(int n, int a, int b)
 {
   if (n == 0) 
-    printf("move stone 1 from %d to %d\n", a, b);
+    printf("presun kamena 1 z %d na %d\n", a, b);
   else
   {
     hanoi(n - 1, a, 6 - a);
-    printf("move stone %d from %d to %d\n", n, a, b);
+    printf("presun kamenna %d z %d nna %d\n", n, a, b);
```
Comment (bwooah) [inline hanoi.c:10]: ajajaj chybaaaa

Comment (annoyingpart) [inline hanoi.c:10]: Carefull
 typo...

Comment (Benko11) [reply hanoi.c:10]: ups :-/

**Review by annoyingpart — CHANGES_REQUESTED** [review]
skuska novy koment

**Pull request merged**